src/main.js

```diff
@@ -138,3 +143,29 @@ const killAllRunningProcesses = () => {
     console.error('Got error when trying to kill children', err);
   }
 };
+
+const showMoveToApplicationsFolderDialog = () => {
+  dialog.showMessageBox(
+    {
+      type: 'question',
+      buttons: ['Yes, move', 'Cancel'],
+      message: 'Move to applications folder?',
+      detail:
+        "I see that I'm not in the Applications folder. I can move myself there if you'd like!",
+      icon: path.join(__dirname, 'assets/icons/png/256x256.png'),
+      defaultId: 0,
+    },
+    res => {
+      if (res === 0) {
+        try {
+          app.moveToApplicationsFolder();
+        } catch (err) {
+          console.error(
+            'Got error when trying to move guppy to the Applications Folder',
+            err
+          );
```

Comment:
Nit: Could we change the copy to `Could not move Guppy to the Applications folder`?

Also, it would be good to call `showMessageBox` again if it fails, with the same message, so that the user is informed.